queries.yaml

```diff
@@ -125,7 +138,6 @@ layers:
     area-inclusion-threshold: 1
   pois:
     template: pois.jinja2
-    start_zoom: 4
```

Comment:
It's always been an undocumented bug/feature that they don't show up earlier. If it turns out to be not ideal we can file a followup issue for it.